Gas Page: More info in After the London Upgrade (#8032)
ethereum/ethereum-org-website: src/content/developers/docs/gas/index.md

Total fee would have been: `Gas units (limit) * Gas price per unit`
i.e `21,000 * 200 = 4,200,000 gwei` or 0.0042 ETH

## After the London upgrade {#post-london}

Let's say Jordan has to pay Taylor 1 ETH. In the transaction, the gas limit is 21,000 units and the base fee is 10 gwei. Jordan includes a tip of 2 gwei. 

> The total fee would now be: `Gas units * (Max fee per gas (Gas Price) + Max priority fee per gas (Tip to the validator))`

Reviewer: not sure about this formnatting tbh because it is n ot clear what is equation and what is text (i.e. a reader could misconstrue the label `(Gas price)` as a multiplication by a parameter named `Gas price` rather than an explanatory label.

```suggestion
The total fee would now be: `units of gas used * (base_fee + priority_fee)` where the `base_fee` is a value set by the protocol and the `priority_fee` is a value set by the user as a tip to the validator.
```

Author: I agree. I was quite hesitant about the details inside the parenthesis, and there was also a mistake with the equation one reason why I drafted this PR haha. Was quite busy but thanks for correcting my mistake!

## After the London upgrade {#post-london}

> Let's say Jordan has to pay Taylor 1 ETH. In the transaction, the gas limit is 21,000 units and the base fee is 10 gwei. Jordan includes a tip of 2 gwei. 

Reviewer 2: ```suggestion
Let's say Jordan has to pay Taylor 1 ETH. In the transaction, the gas limit is 21,000 units and the base fee is 10 gwei. Jordan includes a tip of 2 gwei.
```

The total fee would now be: `units of gas used * (base fee + priority fee)` where the `base fee` is a value set by the protocol and the `priority fee` is a value set by the user as a tip to the validator.

> i.e `21,000 * (10 + 2) = 252,000 gwei` or 0.000252 ETH.

Reviewer 2: ```suggestion

i.e `21,000 * (10 + 2) = 252,000 gwei` or 0.000252 ETH.
```
